src/index.ts

```diff
@@ -82,14 +82,33 @@ export default function supernova(env: Galaxy) {
       const changeSortOrder = useSorting(model, layout.qHyperCube);
 
       const [pageInfo, setPageInfo] = useState(initialPageInfo);
+      const shouldRenderVirtualizedTable = layout.scrollType === 'virtualized';
       const [tableData] = usePromise(
         async () =>
-          env.carbon && !model?.getHyperCubeData
+          env.carbon || shouldRenderVirtualizedTable
```

Comment:
not sure what the consequenses of removing the sanity check for `getHyperCubeData` is here

Reply:
It should not have been remove. Will fix.